.gitignore

```diff
@@ -1,4 +1,4 @@
 /composer.lock
 /vendor
 /tests/tmp
-/tests/.phpunit.result.cache
```

Comment:
Why this change?

Reply:
Because every time I run tests it creates `.phpunit.result.cache` in the root directory instead of `tests`. I have to remove it before I commit. Therefore this should be ignored throughout the whole project.

Reply:
Moved to separate PR for easier review https://github.com/spaze/phpstan-disallowed-calls/pull/54